### ISSUE TITLE
Main xmlcalabash jar file missing

### ISSUE THREAD

**Issue by cmosher01** [ndw/xmlcalabash1#296]
As of release 1.1.28, the main xmlcalabash jar file is missing from the distribution:

```sh
$ unzip -l xmlcalabash-1.1.27-98.zip | grep '/xmlcalabash.*jar'
   953812  2019-06-02 08:04   xmlcalabash-1.1.27-98/xmlcalabash-1.1.27-98.jar
$ unzip -l xmlcalabash-1.1.28-98.zip | grep '/xmlcalabash.*jar'
$ 
```

**Comment by ndw** [comment]
Uhhhmmm. How did...I'm working on it.

**Comment by ndw** [comment]
For some reason, on travis the name of the jar file that's built by gradle is different.

**Comment by chris-mosher** [comment]
I think you should make a `settings.gradle` file, with `rootProject.name = 'xmlcalabash'` in it.

**Comment by ndw** [comment]
That's already in there.

**Comment by cmosher01** [comment]
I was looking at the `saxon99` branch, and I don't see it:
```sh
$ git checkout saxon99
Switched to branch 'saxon99'
Your branch is up to date with 'origin/saxon99'.
$ ls
build.gradle  docs  gradle.properties  lib  README.md  resources  src  test  xpl
```

**Comment by cmosher01** [comment]
```sh
$ gradle dist
...
$ ls build/dist
calabash  calabash.bat  docs  lib  README  schemas  VERSION  xpl


$ echo "rootProject.name = 'xmlcalabash'" >settings.gradle
$ gradle dist
...
$ ls build/dist
calabash  calabash.bat  docs  lib  README  schemas  VERSION  xmlcalabash-1.1.28-99.jar  xpl
```

**Comment by ndw** [comment]
Yeah. That's clearly the problem now, but it used to work and I still have a settings.gradle file in my checked out version, so I'm trying to figure out what went wrong and when (and why).

**Comment by cmosher01** [comment]
If you don't specify the project name in the `settings.gradle` file, it uses the _name_ of the root directory, which can change from system to system, for example if you changed the directory name from `xmlcalabash` to `xmlcalabash1` ;)

**Comment by ndw** [comment]
Yes. I'm going to add the settings.gradle file and push 1.29.x out. What baffles me is how it worked on Travis for 1.27.x! The repo hasn't been renamed recently and the settings.gradle file was, evidently, missing.

**Comment by ndw** [comment]
Resolved in 1.29.x, just released now. Thank you for the bug report and the help!
